metaflow/plugins/metadata/service.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, environment, flow, event_logger, monitor):
     def compute_info(cls, val):
         v = val.rstrip("/")
         try:
-            resp = requests.get(os.path.join(v, "ping"), headers=SERVICE_HEADERS)
+            resp = requests.get(os.path.join(v, "ping"), headers=SERVICE_HEADERS.copy())
```

Comment:
this and other header .copy() calls are a workaround for https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1729 , we can prob remove this once its fixed upstream